python/taichi/lang/__init__.py

```diff
@@ -147,35 +152,35 @@ def init(arch=None,
     env_default_fp = os.environ.get("TI_DEFAULT_FP")
     if env_default_fp:
         if default_fp is not None:
-            core.warn(
+            _ti_core.warn(
                 f'ti.init argument "default_fp" overridden by environment variable TI_DEFAULT_FP={env_default_fp}'
             )
         if env_default_fp == '32':
-            default_fp = f32
```

Comment:
Hmm, not sure if my previous refactoring has already broken this..

Reply:
Not sure how many people use this - it would be nice to make it testable in the future when we have more people :-)